### PR TITLE
Db entities

### PR DESCRIPTION
**Pull request by itshui3**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Fixed entities and submissions post route to work with new entities & ds integration

### CODE REVIEW

**Comment by allie-rae** [comment]
Thank you for doing all this! Story Squad FTW!

File: src/database/entity/Submissions.ts [hunk 1]
```diff
@@ -32,31 +32,31 @@ class Submissions {
...
+    difficult_words: number;
     @Column()
-    linsear_write_formula: string;
+    linsear_write_formula: number;
     @Column()
-    gunning_fog: string;
+    gunning_fog: number;
     @Column()
-    consolidated_score: string;
+    consolidated_score: number;
```
Comment (grifmang) [inline src/database/entity/Submissions.ts]: Should this be string?

File: src/database/entity/Submissions.ts [hunk 2]
```diff
@@ -32,31 +32,31 @@ class Submissions {
...
     @Column()
-    doc_length: string;
+    doc_length: number;
     @Column()
-    quote_count: string;
+    quote_count: number;
     @Column()
-    transcribed_text: string;
+    transcribed_text: number;
```
Comment (grifmang) [inline src/database/entity/Submissions.ts]: Is this an object?

**Review by allie-rae — APPROVED** [review]

File: src/routes/submissions/submissions.routes.ts [hunk 3]
```diff
@@ -1,176 +1,202 @@
...
+        //             // await getRepository(readability, connection()).save({
+        //             //     ...stats,
+        //             //     transcribed_text: story
+        //             // })
+        //             console.log(stats);
+
+        //             return res.status(200).json(stats);
+        //         })
+        //         .catch(console.error);
+        // });
```
Comment (grifmang) [inline src/routes/submissions/submissions.routes.ts:416]: We can probably remove this commented code, so we don't get hit again.

File: src/routes/submissions/submissions.routes.ts [hunk 4]
```diff
@@ -1,176 +1,202 @@
...
+                // // needs to be a sum
+                // linsear_write_formula: readabilityStats[0].linsear_write_formula,
+                // gunning_fog: readabilityStats[0].gunning_fog,
+                // consolidated_score: readabilityStats[0].consolidated_score,
+                // doc_length: readabilityStats[0].doc_length,
+                // // needs to be a sum
+                // quote_count: readabilityStats[0].quote_count,
+                // // needs to be a sum
+                // transcribed_text: readabilityStats[0].transcribed_text,
+                // // needs to be a concatenation
```
Comment (grifmang) [inline src/routes/submissions/submissions.routes.ts:461]: This can be removed as well.

File: src/routes/submissions/submissions.routes.ts [hunk 5]
```diff
@@ -1,176 +1,202 @@
...
+        // cohorts_chapter_id: week,
+        // image: JSON.stringify(data)
+        // })
+
+        // await getRepository(drawing_submissions, connection()).save({
+        //     child_id: req.user.id,
+        //     cohorts_chapter_id: week,
+        //     image: illustration
+        // })
+        // END NEW DB CODE
```
Comment (grifmang) [inline src/routes/submissions/submissions.routes.ts:486]: This can be removed.

File: src/database/entity/Submissions.ts [hunk 6]
```diff
@@ -1,62 +1,64 @@
...
+    @Column({ nullable: true })
+    illustration: string;
+
+    // readability
+    // we will need to make a separate table and link it with individual pages
+    // for now just pass in one page's readability
+    // 3.9.20
+
+    @Column({ type: 'double precision', nullable: true })
+    flesch_reading_ease: number;
```
Comment (grifmang) [inline src/database/entity/Submissions.ts:36]: Ahh, nice.

**Pull request merged**